docker-compose.test.yml

```diff
@@ -101,22 +101,22 @@ services:
       args:
         MPI_KIND: OpenMPI
         PYTHON_VERSION: 2.7
-        TENSORFLOW_PACKAGE: tf-nightly==1.14.1-dev20190408
+        TENSORFLOW_PACKAGE: tf-nightly
         KERAS_PACKAGE: git+https://github.com/keras-team/keras.git
         PYTORCH_PACKAGE: torch-nightly
         PYSPARK_PACKAGE: pyspark==2.4.0
-        MXNET_PACKAGE: mxnet --pre
+        MXNET_PACKAGE: mxnet==1.5.0b20190412 --pre
```

Comment:
--pre can be removed. Same for other places.

Reply:
I wanted to keep it to not forget it once we remove specific version tag.

Reply:
sounds good.